beacon_node/src/config.rs

```diff
@@ -645,6 +645,10 @@ pub fn get_config<E: EthSpec>(
         client_config.chain.enable_lock_timeouts = false;
     }
 
+    if cli_args.is_present("disable-eth1-sync") {
```

Comment:
It would be awesome if you could add tests in `lighthouse/tests/beacon_node.rs` that test the default setting (with no flag) and verification of `sync_eth1_chain == false` when the flag is provided.

Reply:
✅

Reply:
thanks!